### ISSUE TITLE
about true_boxes in YoloLayer

### ISSUE THREAD

**Issue by muye5** [experiencor/keras-yolo3#71]
1. I cannot understand why `true_boxes` need to be passed into YoloLayer. 
`conf_delta *= tf.expand_dims(tf.to_float(best_ious < self.ignore_thresh), 4)`
this line filter out some elements which match `true_boxes`, but how it effect the loss, 
`conf_delta = object_mask * (pred_box_conf - true_box_conf) * self.obj_scale + (1 - object_mask) * conf_delta * self.noobj_scale`

2. why normalizing of loss missed in yolo3. not same with yolo2.
`loss_xy = tf.reduce_sum(tf.square(xy_delta), list(range(1, 5)))`

thanks!

**Comment by clive819** [comment]
1. You might want to take a look at [section 2.1](https://pjreddie.com/media/files/papers/YOLOv3.pdf).
2. I'm confused too!

**Comment by clive819** [comment]
Elements that does overlap a ground truth object by more than **self.ignore_thresh** will be zero, so they won't be included in **loss_conf**. 

> If the bounding box prior is not the best but does overlap a ground truth object by more than some threshold we ignore the prediction, following [17].

**Comment by muye5** [comment]
@clive819  got it. thank you very much.
